### DISCUSSION TITLE
Data Engineering

### DISCUSSION THREAD

**Discussion by sudipbhattarai820** [ravi7522/Football-Prediction#23] [General]
Do we have any new clean data??

**Comment by babuaditya97** [comment]
Hi sudip. we got one update today. I'm currently doing the evaluations on that. Also, open question .
Can we move the part in the model training and evaluation file(football_prediction.ipynb) which does the missing value handling to the data preprocess file? so that only training, tuning and evaluations are there.
what do you guys think
